components/PostCard.tsx

```diff
@@ -1,13 +1,35 @@
+import {
+  isPostADraft,
+  isPostInTheFuture,
+} from "helpers/checkOfDraftOrFuturePost";
 import { PostDocumentWithoutContent } from "interfaces";
 import Image from "next/image";
 import Link from "next/link";
 import { useAppDispatch } from "redux/typesHooks";
 
 import { setTags } from "../redux/slices/selectedTags";
+
 interface Props {
   post: PostDocumentWithoutContent;
 }
 
+export const DraftPostMark = () => (
+  <div className="draft-post">
+    <div className="triangle-draft triangle triangle-item">
```

Comment:
@ntorbinskiy 

Start class, with the main class triangle, and then add modificators

```suggestion
    <div className="triangle triangle-draft  triangle-item">
```